doc/api/child_process.markdown

```diff
@@ -160,7 +162,7 @@ to be an error.
 The `options` argument may be passed as the second argument to customize how
 the process is spawned. The default options are:
 
-```js
+```json
```

Comment:
Be aware that changing the markdown from `js` to `json` requires a valid JSON object, with all the keys and string values having double quotes.